docs/source/transaction_freeze_ledgers.md

```diff
@@ -2,7 +2,7 @@
 
 #### Why It May Be Needed
 
-- Removing of deprecated ledgers need of painless mechanism. Transaction freeze ledgers give a security transition. Transaction Freeze Ledgers disable specific ledgers (system ignore frozen ledgers).
+- Removing of deprecated ledgers need of painless mechanism. Transaction freeze ledgers give a security transition. Transaction Freeze Ledgers disable plugin's ledgers (frozen ledger can be used neither for reading not for writing).
```

Comment:
We can reed from a frozen ledger (!) but after removing it via remove_ledger.py, this is impossible.